scripts/clean-build.sh

```diff
@@ -40,14 +40,24 @@ apt-get clean
 
 cd /anbox
 
-# In cases where anbox comes directly from a checked out Android
-# build environment we miss some symlinks which are present on
-# the host and don't have a valid git repository in that case.
-git clean -fdx . || true
-git reset --hard || true
+cleanup() {
+  # In cases where anbox comes directly from a checked out Android
+  # build environment we miss some symlinks which are present on
+  # the host and don't have a valid git repository in that case.
+  git clean -fdx . || true
```

Comment:
Could we be more strict here and check if we have a .git for example? || true might shadow actual issues over time.